source-kinesis/connection.go

```diff
@@ -43,30 +43,31 @@ var configJSONSchema = `{
 		"awsSecretAccessKey"
 	],
 	"properties": {
-		"region": {
-			"type":        "string",
-			"title":       "AWS Region",
-			"description": "The name of the AWS region where the Kinesis stream is located",
-			"default":     "us-east-1"
-		},
-		"endpoint": {
-			"type":        "string",
-			"title":       "AWS Endpoint",
-			"description": "The AWS endpoint URI to connect to, useful if you're capturing from a kinesis-compatible API that isn't provided by AWS"
-		},
 		"awsAccessKeyId": {
 			"type":        "string",
 			"title":       "AWS Access Key ID",
 			"description": "Part of the AWS credentials that will be used to connect to Kinesis",
-			"default":     "example-aws-access-key-id",
-			"secret": true
```

Comment:
This doesn't need to be a secret. Also it doesn't seem helpful to have a default value here since `example-aws-access-key-id` is never going to be the correct value.